explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -52,44 +53,59 @@ object AladinCell extends ModelOptics {
 
   implicit val propsReuse: Reusability[Props] = Reusability.derive[Props]
 
+  val angleFovLens: Lens[Angle, Fov] = Lens[Angle, Fov](a => Fov(a, a))(f => (_ => f.x))
+
+  val fovL = TargetVisualOptions.fov.some.andThen(angleFovLens)
+
   val component =
     ScalaFnComponent
       .withHooks[Props]
       // mouse coordinates, starts on the base
       .useStateBy(_.aladinCoords)
-      // field of view
-      .useStateViewWithReuseBy((p, _) => Fov(p.options.get.fovAngle, p.options.get.fovAngle))
+      // target options, will be read from the user preferences
+      .useStateViewWithReuse(TargetVisualOptions.Default)
       // flag to trigger centering. This is a bit brute force but
       // avoids us needing a ref to a Fn component
       .useStateViewWithReuse(false)
-      .renderWithReuse { (props, mouseCoords, fov, center) =>
+      .useEffectWithDepsBy((p, _, _, _) => (p.uid, p.tid)) { (props, _, options, _) => _ =>
+        implicit val ctx = props.ctx
+        UserTargetPreferencesQuery
+          .queryWithDefault[IO](props.uid, props.tid, Constants.InitialFov)
+          .flatMap(fov => options.mod(_.copy(fov = fov.some)).to[IO])
+          .runAsyncAndForget
```

Comment:
If you `runAsync` instead (which requires a `Logger` but we already have that in context here), it will log any errors. Otherwise they will be silenced.